jenkins/Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+pipeline {
+  agent {
+    kubernetes {
+      label 'ci'
+      defaultContainer 'jnlp'
+      yamlFile 'jenkins/build-pod.yaml'
+    }
+  }
+
+  options {
+    checkoutToSubdirectory('cloudify-java-rest-client')
+    buildDiscarder(logRotator(numToKeepStr:'30'))
+    timeout(time: 60, unit: 'MINUTES')
+    timestamps()
+  }
+
+  environment {
+    branch = "master"
+    workspace = "${env.WORKSPACE}"
+    target_dir = "cloudify-java-rest-client-${env.BUILD_NUMBER}"
+    cfy_manager_url = 'https://raw.githubusercontent.com/cloudify-cosmo/cloudify-java-rest-client'
```

Comment:
better to call it cloudify-java-rest-client_url

Reply:
done